package.json

```diff
@@ -19,7 +19,7 @@
         "e2e:ui": "playwright test tests/e2e --ui"
     },
     "dependencies": {
-        "@appwrite.io/console": "https://pkg.pr.new/appwrite-labs/cloud/@appwrite.io/console@c985738",
+        "@appwrite.io/console": "1.6.0",
```

Comment:
we sholdn't revert this as it contains the new response model of the regions endpoint with disabled and available attributes